duden/cli.py

```diff
@@ -114,6 +117,8 @@ def parse_args():
 
     parser.add_argument('-V', '--version', action='store_true',
                         help=_('print program version'))
+    parser.add_argument('--phonetic', action='store_true',
```

Comment:
One more thing, could you please alse add the `--phonetic` switch to shell completion scripts?
https://github.com/radomirbosak/duden/blob/master/completions/duden
https://github.com/radomirbosak/duden/blob/master/completions/duden.fish